### ISSUE TITLE
Vulkan-LoaderAndValidationLayers does not provide pkg-config file ( ex. vulkan.pc) 

### ISSUE THREAD

**Issue by elongbug** [KhronosGroup/Vulkan-LoaderAndValidationLayers#757]
current Vulkan-LoaderAndValidationLayers does not provide pkg-config file.
in linux environment, Makefile usually uses pkg-config tools for setting  name of library linking. 
at this point of time we should describe library linking information like -lvulkan. 
so for convenience of vulkan application, we need to provide vulkan.pc .

**Comment by johnzupin** [comment]
Do you still need this? The SDK no longer installs resources to system paths.

**Comment by elongbug** [comment]
Yes, I need still this feature. In linux envirionment,  Vulkan-LoaderAndValidationLayers  provides vulkan library and header files,  so they also provides package-config file.

**Comment by karl-lunarg** [comment]
We understand now that this isn't about the SDK. 

Is it about using a vulkan package-config file to locate include and headers built within the user's Vulkan-LoaderAndValidationLayers repo?

Or,

Are you a package maintainer that wants to add a vulkan package to a linux distro and wants us to supply a vulkan.pc file to include in your distro?

If the first case, a developer using the Vulkan-LoaderAndValidationLayers repo, we see that there is support in CMake for generating a vulkan.pc file.  This is something that we could look into adding soon.  Is this what you want?

Generating a vulkan.pc file seems most useful (to pkg-config) if this file is placed someplace like /usr/lib/pkg-config.  A regular CMake build won't put the vulkan.pc file there.  Note that we've recently added "make install" support that puts the vulkan libraries in system directories.  Would you like "make install" to also install vulkan.pc in /usr/lib/pkg-config?

To summarize, could you please answer the following:

1) Are you asking about this from the point of view of a package maintainer or a developer using this Vulkan-LVL repo?  In other words, please explain your use case.

2) Where would you like the vulkan.pc file to be placed when it is generated and how do you plan to tell pkg-config where to find it?  Do you expect vulkan.pc to be in system dirs, or do you plan to use an env var to tell pkg-config where to find it?

**Comment by strassek** [comment]
I'm not sure if I understand the dilemma. It seems sensible to install vulkan.pc to ${prefix}${libdir}/pkg-config as is the case with most Linux packages. That ought to cover distro maintainers. People like me, who want to build Vulkan-LVL from source and install to an alternate preifix, are free to use the PKG_CONFIG_PATH env var accordingly.

**Comment by karl-lunarg** [comment]
I started working on this, but I have not tested the install targets yet.

If you ( @strassek or @elongbug ) would like to take a look, please see the karl-pkgconfig branch in this repo.
